README.md

```diff
@@ -1,3 +1,56 @@
 # Bichard 7 Core
 
 The code to replace the processing logic of Bichard 7
+
+## Testing
+
+To run unit tests against new Bichard:
+
+```bash
+npm i # If packages not already installed
+npm t
+```
+
+To run unit tests against old Bichard (*requires the [old Bichard](https://github.com/ministryofjustice/bichard7-next) stack to be running*):
+
+```bash
+npm i # If packages not already installed
+npm run test:bichard
+```
+
+## Comparing New and Old Bichard
+
+Old Bichard is currently configured to publish its outputs (AHO, triggers, exceptions), plus the original message it received to the `PROCESSING_VALIDATION_QUEUE` queue in ActiveMQ. We can compare these outputs against new Bichard by running:
+
+```bash
+npm i # If packages not already installed
+npm run compare
+```
+
+This requires the [old Bichard](https://github.com/ministryofjustice/bichard7-next) stack to be running. Outputs from old Bichard can be driven onto the queue by running either `make pushq` in the [old Bichard repo](https://github.com/ministryofjustice/bichard7-next), or by running `npm t` in the [e2e testing repo](https://github.com/ministryofjustice/bichard7-next-tests).
+
+This will run the `scripts/compareResults.ts` script to check the output of old Bichard against new Bichard. A tally is kept of the results and can be seen by exiting the script with `Ctrl-C`. If `compareResults.ts` can't process any messages off the queue (empty messages, ActiveMQ errors ect), these are recorded as `skipped` and will be counted seperatly.
```

Comment:
Can you make a little more explicit that it's the npm run compare cmd that runs `compareResults.ts`. My first reading of it led me to think that the previous para's cmds ran the script

Reply:
Yep! Done :)